src/tiff.imageio/CMakeLists.txt

```diff
@@ -4,5 +4,4 @@
 
 add_oiio_plugin (tiffinput.cpp tiffoutput.cpp
                  INCLUDE_DIRS ${TIFF_INCLUDE_DIR}
```

Comment:
Can this TIFF_INCLUDE_DIR go away since you're using the TIFF::TIFF target ?

Reply:
Yes, it has been removed.